src/distribution/binomial.rs

```diff
@@ -319,26 +319,23 @@ fn approximate_by_normal(p: f64, np: f64, v: f64, u: f64) -> f64 {
     let p3 = p2 * p;
     let p4 = p2 * p2;
 
-    np +
-    sd * w +
-    (p + 1.0) / 3.0 -
-    (2.0 * p - 1.0) * w2 / 6.0 +
-    sd_em1 * w3 * (2.0 * p2 - 2.0 * p - 1.0) / 72.0 -
-    w * (7.0 * p2 - 7.0 * p + 1.0) / 36.0 +
-    sd_em2 * (2.0 * p - 1.0) * (p + 1.0) * (p - 2.0) * (3.0 * w4 + 7.0 * w2 - 16.0 / 1620.0) +
-    sd_em3 * (
-        w5 * (4.0 * p4 - 8.0 * p3 - 48.0 * p2 + 52.0 * p - 23.0) / 17280.0 +
-        w3 * (256.0 * p4 - 512.0 * p3 - 147.0 * p2 + 403.0 * p - 137.0) / 38880.0 -
-        w * (433.0 * p4 - 866.0 * p3 - 921.0 * p2 + 1354.0 * p - 671.0) / 38880.0
-    ) +
-    sd_em4 * (
-        w6 * (2.0 * p - 1.0) * (p2 - p + 1.0) * (p2 - p + 19.0) / 34020.0 +
-        w4 * (2.0 * p - 1.0) * (9.0 * p4 - 18.0 * p3 - 35.0 * p2 + 44.0 * p - 25.0) / 15120.0 +
-        w2 * (2.0 * p - 1.0) * (
-                923.0 * p4 - 1846.0 * p3 + 5271.0 * p2 - 4348.0 * p + 5189.0
-        ) / 408240.0 -
-        4.0 * (2.0 * p - 1.0) * (p + 1.0) * (p - 2.0) * (23.0 * p2 - 23.0 * p + 2.0) / 25515.0
-    )
+    np + sd * w + (p + 1.0) / 3.0 - (2.0 * p - 1.0) * w2 / 6.0
```

Comment:
Answering your question, Rustfmt is mandatory. Well, except for a few cases 😀 It’s just that some blocks of code become extremely illegible once formatted, and so I usually cherry-pick rustfmt’s changes. In this case, I think it’s all right. However, I wouldn’t mix formatting changes with conceptual changes. If you don’t mind, at least create a separate commit, and I won’t squash.